packages/api/src/apps/api/app.ts

```diff
@@ -189,6 +179,8 @@ export default async (env: ProcessEnv) => {
     // Should switch all clients to explicit channels
     ["emp", Actions.Emp(undefined, appState)],
     ["lsp", Actions.Lsp(undefined, appState)],
+    // TODO: switch this to root path once frontend is ready to transition
+    ["global", Actions.Global(undefined, appState)],
```

Comment:
this adds the `/global` channel, i dont want to inadvertently break production until francesco is ready to update api